packages/react/package.json

```diff
@@ -50,7 +50,7 @@
     "react-hook-form": "^7.38.0",
     "react-router-dom": "^6.3.0",
     "typescript": "^4.5.2",
-    "webpack": "^5.75.0"
+    "webpack": "^5.76.0"
```

Comment:
When I ran audit fix in my local environment, it gave me this version: "5.76.2"

Reply:
Was going to say something similar, I done a audit fix on ic-design-system and it updated it to 5.76.1 (https://github.com/mi6/ic-design-system/pull/341/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519R28325). If we want to be explicit, I guess we can update the version however, a new install will give us the latest minor/patch version anyway.